tests/test_instrument_tests.py

```diff
@@ -148,15 +147,14 @@ def test_sort(self):
 import unittest
 
 import dill as pickle"""
-    if platform.system() != "Windows":
-        imports += "\nimport timeout_decorator"
+    # timeout_decorator no longer used since pytest handles timeouts
 
     imports += "\n\nfrom code_to_optimize.bubble_sort import sorter"
     
     wrapper_func = codeflash_wrap_string
     
     test_class_header = "class TestPigLatin(unittest.TestCase):"
-    test_decorator = "    @timeout_decorator.timeout(15)" if platform.system() != "Windows" else ""
+    test_decorator = ""  # pytest-timeout handles timeouts now, not timeout_decorator
```

Comment:
i would ask you to double check this assumption. i remember there were problems earlier. just confirm this point